projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -1404,3 +1412,37 @@ public static function register_block_metadata_collection() {
 	 */
 	add_filter( 'jetpack_block_editor_enable_upgrade_nudge', '__return_true' );
 }
+
+/**
+ * Overrides the `wp-block-editor` script with a hotfix to prevent a performance issue.
+ *
+ * @param WP_Scripts $scripts WP_Scripts instance.
+ */
+function jetpack_hotfix_block_editor_script( $scripts ) {
+	if ( ! defined( 'GUTENBERG_VERSION' ) ) {
+		return;
+	}
+
+	if ( ! in_array( GUTENBERG_VERSION, array( '19.9.0', '20.0.0', '20.1.0' ), true ) ) {
+		return;
+	}
+
+	$path         = gutenberg_dir_path() . 'build/block-editor/index.min.js';
```

Comment:
Phan isn't too happy about this, so you'll probably have to make an update to the config.

Reply:
Added Gutenberg stubs in https://github.com/Automattic/jetpack/pull/41309/commits/f6c40cca183fee755ce88834ab70447a18131414